apps/src/javalab/ControlButtons.jsx

```diff
@@ -4,76 +4,101 @@ import i18n from '@cdo/locale';
 import color from '@cdo/apps/util/color';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 import JavalabButton from './JavalabButton';
+import JavalabSettings from './JavalabSettings';
 
 export default function ControlButtons({
-  isDarkMode,
   isRunning,
   isTesting,
   toggleRun,
-  toggleTest
+  toggleTest,
+  isEditingStartSources,
+  isReadOnlyWorkspace,
+  onContinue,
+  renderSettings
 }) {
-  const buttonStyles = {
-    ...styles.button.all,
-    ...(isDarkMode ? styles.button.dark : styles.button.light)
-  };
-
   // Note: The 'noBorder' class is required on the buttons below because there are !important
   // button styles we don't want. This class can be removed when the button:active !important
   // border is removed from common.scss.
   return (
-    <div
-      style={{
-        ...styles.container,
-        backgroundColor: isDarkMode ? color.black : color.white
-      }}
-    >
+    <div>
       <JavalabButton
```

Comment:
Brendan, you and @bethanyaconnor may want to coordinate these changes as she's also making some changes in this area for contained levels.